source/content/global-cdn.md

```diff
@@ -32,6 +32,57 @@ We recommend installing the Pantheon Advanced Page Cache [plugin](https://wordpr
 
 For more details, see [Clearing Caches for Drupal and WordPress](/clear-caches).
 
+## Persistent Cache
+
+Serve your Drupal or WordPress site even in the unlikely event that it goes down.
+
+If the server is not responding and can't serve a new copy of a page, a cached version will be favored over displaying an error, even if the cached version has expired (this is called _stale cache_). With Persistent Cache, the goal is to provide a seamless, uninterrupted experience for the user.
+
+### Caching Exceptions - How long does content stay fresh?
+
+Adjust the length of time before the site's cached content is considered stale by adjusting the time-to-live (TTL).
+
+Your site’s CMS page-level caching must be correctly configured in order to take advantage of Persistent Cache.
+
+On [Drupal](/drupal-cache#drupal-8-performance-configuration) and [WordPress](/wordpress-cache-plugin#pantheon-page-cache-plugin-configuration), you can adjust your CDN edge configuration to serve stale content for a specific amount of time.
+
+For best results, set the cache TTL to a value equal to or over 3700 seconds.
+
+Users with session-style cookies set, or a `NO_CACHE` cookie set will bypass the cache, and will not see cached content. For best results, set the `NO_CACHE` cookie to persist longer than the site’s page cache (this includes logged in users and authenticated traffic). You can learn more about the exceptions to page caching rules in [Caching: Advanced Topics](/caching-advanced-topics#allow-a-user-to-bypass-the-cache).
+
+### Confirm That Persistent Cache Works
+
+To test how stale cache is served, compare the header results of a page refresh of the site's Dev environment to the same page in Maintenance Mode.
+
+Navigate to the page using [Firefox](https://developer.mozilla.org/en-US/docs/Tools) or [Chrome](https://developer.chrome.com/docs/devtools/), and in the browser's developer tools open the **Network** tab and view the response headers for the page or asset. Refresh the page in the browser for the newest information.
+
+Examine the results for the `age` and `max-age`.
+
+To examine the headers through the command line, use a `curl` command to examine the headers before and after you set the site to Maintenance Mode:
+
+```bash{outputLines: 2-20}
+curl --head https://pantheon.io/docs
+HTTP/2 301
+content-type: text/html
+location: https://pantheon.io/docs/
+server: nginx
+strict-transport-security: max-age=31622400
+x-pantheon-styx-hostname: styx-fe2-a-5d96768699-vcdvh
+x-styx-req-id: b7b8d4d2-04d9-11ec-a467-9a05fab906d1
+cache-control: public, max-age=86400
+date: Tue, 24 Aug 2021 15:30:21 GMT
+x-served-by: cache-mdw17379-MDW, cache-ewr18124-EWR
+x-cache: HIT, HIT
+x-cache-hits: 1, 1
+x-timer: S1629819022.932985,VS0,VE1
+pantheon-trace-id: be58e6a03a904fbfa64515ee136ffd34
+vary: Cookie, Cookie
+age: 9654
+accept-ranges: bytes
+via: 1.1 varnish, 1.1 varnish
+content-length: 162
+```
```

Comment:
I'm not super clear on which would be the best header response to look out for, so this whole set of lines should get reviewed by @mbaynton and @mattcallahan and then we can either add a `//highlight-start` and `-end` to show people what to look out for in the output, or `grep` for a keyword if that's easy enough

Reply:
I don't think this response actually demonstrates the feature. This response looks like a healthy cache hit. Also pantheon.io specifically redirects all of the time when you're using `cURL` for some reason, I've noticed this as well but am unsure what's causing it.
To see the feature operating, one must also prevent Fastly's regular old fresh cache for the page from being used. There's basically two ways to do that, either wait more than the `max-age` (86400s here, not fun, may not work anyway since the cache, you know, isn't persistent) or purge the cache, either for the page using for example the Advanced Page Cache module or for the whole site from the dashboard.
Once you've done that, you should see an additional header in the response, `PContext-Resp-Is-Stale: true`, which indicates the reason a non-error response was returned was because of this feature.

Reply:
thanks, @mbaynton! you're right - this snippet doesn't show staled (the text above it just introduces it as a way to view headers) - but that `PContext-Resp-Is-Stale: true` is perfect. I can add another example bash command like

```
curl --head https://pantheon.io/docs | grep PContext-Resp-Is-Stale
```

to this point:

> either wait more than the max-age (86400s here, not fun, may not work anyway since the cache, you know, isn't persistent) or purge the cache, either for the page using for example the Advanced Page Cache module or for the whole site from the dashboard.

should the steps to test this be more like (adding step 2):
1. maint mode
2. clear cache either from advanced page cache or dash
3. check headers

Reply:
@mbaynton - All updated, if you want to take a look 👀